upstairs/src/block_io.rs

```diff
@@ -233,6 +241,14 @@ impl BlockIO for ReqwestBlockIO {
         unimplemented!();
     }
 
+    fn write_unwritten(
+        &self,
+        _offset: Block,
+        _data: Bytes,
+    ) -> Result<BlockReqWaiter, CrucibleError> {
+        unimplemented!();
```

Comment:
`CrucibleError::Unsupported` instead?

Reply:
Updated this and the one above it (free unrelated change)